### PR TITLE
Jordi/cf 14 set up playwright

### PR DESCRIPTION
**Pull request by meyerjrr**
Playwright set up with auth injection
next steps 
- will be writing the actual tests
- CI 
Because of how Playwright sets up the user auth. Will need to work out how to write tests for users and admins separately. Can spec this up in a notion tomorrow.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -53,12 +54,16 @@
     "@types/react-dom": "18.0.5",
     "@types/react-table": "^7.7.11",
     "@vitejs/plugin-react": "^1.3.2",
+    "aws-amplify": "^4.3.27",
```
Comment (chrnorm) [inline web/package.json]: can you remove this import? We have `@aws-amplify/auth` which you can use instead!

File: web/package.json [hunk 2]
```diff
@@ -53,12 +54,16 @@
...
+    "dotenv": "^16.0.1",
     "eslint": "8.13.0",
     "eslint-config-next": "12.1.5",
+    "fs": "0.0.1-security",
```
Comment (chrnorm) [inline web/package.json]: `fs` is built in to Node so you shouldn't need to install it - we can remove this

**Review by chrnorm — DISMISSED** [review]
quick couple of dependency suggestions from me

File: web/package.json (hunk 2 above)
Comment (jordiup) [reply web/package.json]: Cheers Chris, amended

**Comment by jordiup** [comment]
Going to merge this one in as our Desired Outcomes are met for this initial ticket :)

**Review by jordiup — APPROVED** [review]

**Pull request merged**